### DISCUSSION TITLE
What to do with taxa that have multiple trophic modes or guilds?

### DISCUSSION THREAD

**Discussion by brendanf** [brendanf/FUNGuildR#4] [Q&A]
@yuwingjane asked this in the thread of a different question:
>I have been confused by a question, a taxon is assigned to more than one trophic mode, such as Pathotroph-Saprotroph-Symbiotroph and the confidenceRanking is highly probably. Does this mean that this taxon can perform all these trophic modes? But which trophic mode should I assign this taxon to? Pathogen, saprotroph, or symbiotroph?

**Comment by brendanf** [comment]
This can happen with trophic modes, guilds, growth forms, and traits.

In general this can mean either two things:

1) The organism really does have multiple trophic modes (guilds, growth forms, traits...), for instance at different life stages or under different conditions; or
2) We don't have information about that organism in particular, and we are relying on information about its genus, family, or order, and so there are several possibilities.

In my own analyses, the way I treat these depends on what question I am asking. If I am trying to sum up the fraction of each trophic mode, I usually treat a "Pathotroph-Saprotroph-Symbiotroph" as 1/3 pathotroph, 1/3 saprotroph, 1/3 symbiotroph.  If I am specifically looking at a particular trophic mode, for instance Saprotroph, then I will choose a cutoff confidence ranking, often "Probable", and include all of the taxa which include Saprotroph as one of the possibilities.

**Reply by yuwingjane** [reply]
Hi Brendanf, thank you very much! Your reply is very helpful:)
